packages/preferences/src/browser/views/preference-widget-bindings.ts

```diff
@@ -44,15 +51,37 @@ function createPreferencesWidgetContainer(parent: interfaces.Container): Contain
     child.bind(PreferencesTreeWidget).toSelf();
     child.rebind(TreeProps).toConstantValue({ ...defaultTreeProps, search: false });
     child.bind(PreferencesEditorWidget).toSelf();
-    child.bind(PreferencesDecorator).toSelf();
-    child.bind(PreferencesDecoratorService).toSelf();
-    child.rebind(TreeDecoratorService).toService(PreferencesDecoratorService);
 
-    child.bind(SinglePreferenceWrapper).toSelf();
     child.bind(PreferencesSearchbarWidget).toSelf();
     child.bind(PreferencesScopeTabBar).toSelf();
-    child.bind(SinglePreferenceDisplayFactory).toSelf();
     child.bind(PreferencesWidget).toSelf();
 
+    child.bind(PreferenceNodeRendererFactory).toFactory(({ container }) => (node: Preference.TreeNode) => {
+        const grandChild = container.createChild();
```

Comment:
Since `child` is already defined in the outer scope, calling this `child` as well would produce a lint error. Now whether it should be camel-cased is open for discussion... 😅

Reply:
Oh no ok didn't see the names before that, my bad.